Source/Relativity.Transfer.SDK.Samples.Repository/FullPathWorkflow/DownloadDirectory.cs

```diff
@@ -41,6 +42,12 @@ public async Task ExecuteAsync(Configuration configuration, CancellationToken to
             : new DirectoryPath(configuration.DownloadDirectory.Source);
         var destination = _pathExtension.EnsureLocalDirectory(configuration.DownloadDirectory.Destination);
         var authenticationProvider = _relativityAuthenticationProviderFactory.Create(configuration.Common);
+        var downloadDirectoryOptions = new DownloadDirectoryOptions()
+        {
+            MaximumSpeed = default,

```

Comment:
why we added everywhere default for MaxSpeed and OverwritePolicy ? 

Shouldn't we have separate sample scenario for showing that transferspeed is limited?

Reply:
I think this is only example that we have options objects and client can configure it. Because before clients were not aware to pass this options to transfer. We have more properties in these options objects, so if you want I can add new examples with each option. Do we really need it?